### PR TITLE
Add meta docs

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This adds a README and notes a license for this repo.

cc @richardbarlow and @prophile as other authors of the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * [Battery](src/battery.md)
+ * [Motor Board](src/motor-board.md)
+ * [Odroid](src/odroid.md)
+ * [Power Board](src/power-board.md)
+ * [Ruggeduino](src/ruggeduino.md)
+ * [Servo Board](src/servo-board.md)
+ * [Usb Charger](src/usb-charger.md)
+ * [Usb Hub](src/usb-hub.md)
+
+## License
```
Comment (RealOrangeOne) [inline README.md]: We should add a dedicated `LICENSE` file as well, as with https://github.com/srobo/runbook/blob/master/LICENSE

**Comment by richardbarlow** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (PeterJCLaw) [reply README.md]: 03a0248bbb8c410e4e4b944fa17876fbdc34abd0 does this.

**Comment by PeterJCLaw** [comment]
@richardbarlow are you able to merge this? (I don't seem to have permission to do so)

**Pull request merged**